app/src/main/res/layout/admin_pin_activity.xml

```diff
@@ -10,105 +10,96 @@
       type="org.oppia.android.app.profile.AdminPinViewModel" />
   </data>
 
-  <FrameLayout
+  <androidx.core.widget.NestedScrollView
+    android:id="@+id/scrollViewAdminPin"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
-
-    <ScrollView
-      android:id="@+id/scrollViewAdminPin"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
leave a single line above

Reply:
Done

Reply:
@bladebunny please let the author resolve, you can just comment and the author will check the required changes and mark it resolved.